plugins/janus_streaming.c

```diff
@@ -1815,6 +1816,18 @@ struct janus_plugin_result *janus_streaming_handle_message(janus_plugin_session
 		json_object_set_new(ml, "description", json_string(mp->description));
 		json_object_set_new(ml, "type", json_string(mp->streaming_type == janus_streaming_type_live ? "live" : "on demand"));
 		json_object_set_new(ml, "is_private", mp->is_private ? json_true() : json_false());
+		if(mp->streaming_source == janus_streaming_source_rtp) {
```

Comment:
While testing I noticed this activates when creating RTSP mountpoints too, which shouldn't happen as in that case it's the RTSP server that feeds those addresses automatically, and there's no need to spawn scripts yourselves. I think an easy way to fix that is to change the check you added to something like `if(!strcasecmp(type_text, "rtp")) {` instead, as this way we're sure it only triggers for plain RTP mountpoints. An alternative might be accessing the `source` property within this check, cast it to `janus_streaming_rtp_source`, and only return the ports if the `rtsp` boolean property is `FALSE` (notice that, in case, this check would need to be wrapped in a `#ifdef HAVE_LIBCURL`, as that boolean wouldn't exist otherwise).